apps/namadillo/src/utils/compatibility.tsx

```diff
@@ -0,0 +1,94 @@
+import { routes } from "App/routes";
+import compatibilty from "compatibility.json";
+import { wallets } from "integrations";
+import { Link } from "react-router-dom";
+import semverSatisfies from "semver/functions/satisfies";
+import semverLtr from "semver/ranges/ltr";
+
+// Checks if the versions are compatible using semantic versioning (semver).
+// Returns true if the versions are compatible, -1 if the version is outdated,
+// or 1 if a required version is lower than the version provided.
+const checkVersionsCompatible = (
+  currentVersion: string,
+  requiredVersion: string
+): -1 | 1 | boolean => {
```

Comment:
It's a personal taste, but don't you prefer an `enum` or `const` here? it's hard to read later on the conditions below